src/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.form {
+  position: relative;
+  margin: 20px 8px;
+}
+
+.search-bar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
```

Comment:
Move to `*` selector and remove in all other places.

Reply:
Do you mean box-sizing?